tests/fixtures/routes/routes-tuples.php

```diff
@@ -0,0 +1,5 @@
+
+
+Route::resource('foo', App\Http\Controllers\FooController::class);
```

Comment:
Unfortunately, this is not (currently) allowed unless the `RouteServiceProvider::$namespace` property is empty.

Reply:
There could be a note about this in the config file.